inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -2008,7 +2008,11 @@ void MKLDNNInterpolateNode::createPrimitive() {
     jcp.ID = srcDimPad5d[2];
     jcp.spatial_dim_size = spatialDimSize;
 
-    if (getChildEdgeAt(0)->getMemory().GetDesc().isPlainFormat()) {
+    // TODO: the result of isPlainFormat() methods for child edges is currently incorrect in inplace cases,
+    //  because of strides overriding
+    if ([&]() { auto conf = getSelectedPrimitiveDescriptor()->getConfig().outConfs[0];
+        return conf.inPlace ? (conf.desc.getLayout() == NCHW || conf.desc.getLayout() == NCDHW):
+               getChildEdgeAt(0)->getMemory().GetDesc().isPlainFormat();}()) {
```

Comment:
This layout tags are considered to be unreliable  in the CPU plugin context. I suggest using the comparison of the result of `PartialBlkDesc::makePlain` with the result of function `PartialBlkDesc::extractFrom` to reliably check if the layout is plain.

Reply:
Applied